index.d.ts

```diff
@@ -428,6 +431,10 @@ declare namespace Shopify {
         max: number;
     }
 
+    interface IAccessScope {
+        handle: string
```

Comment:
I think we can define a type with all [scopes](https://help.shopify.com/api/getting-started/authentication/oauth/scopes) and use it here instead of just `string` but I'm not sure if it makes sense.